app/src/main/java/net/gsantner/markor/ui/SearchOrCustomTextDialogCreator.java

```diff
@@ -125,8 +125,15 @@ public static void showInsertTableRowDialog(final Activity activity, final boole
 
     public static void showSearchFilesDialog(Activity activity, File searchDir, Callback.a2<String, Integer> callback) {
         if (!SearchEngine.isSearchExecuting) {
+            final char[] password;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M) {
+                password = new AppSettings(activity).getDefaultPassword();
```

Comment:
please have the assignment in FileSearchDialog.java, there we also assign other AppSettings default values

![grafik](https://user-images.githubusercontent.com/6735650/125639263-7c6442c1-598f-4b31-b72f-918e3adf7a55.png)